introduction.md

```diff
@@ -1,2 +1,5 @@
 Hey I'm Geraud and I'm trying out github
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?